keystrokes/src/components/Character/CharacterCreationModal.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState} from 'react';
+import {Modal, Button, TextField, FormControl, InputLabel, Select, MenuItem, Grid} from '@material-ui/core';
```

Comment:
You're spacing is a bit inconsistent here, not the end of the world but prettier would help make it consistent.